torch/utils/data/datapipes/iter/filelister.py

```diff
@@ -1,22 +1,24 @@
+from typing import Iterator, List, Sequence, Union
+
 from torch.utils.data import IterDataPipe
+from torch.utils.data.datapipes.iter import IterableWrapper
 from torch.utils.data.datapipes.utils.common import get_file_pathnames_from_root
-from typing import List, Union, Iterator
 
 class FileListerIterDataPipe(IterDataPipe[str]):
     r""" :class:`FileListerIterDataPipe`
 
     Iterable DataPipe to load file pathname(s) (path + filename), yield pathname from given disk root dir.
 
     Args:
-        root: Root directory
+        root: Root directory or a sequence of root directories
         mask: Unix style filter string or string list for filtering file name(s)
         abspath: Whether to return relative pathname or absolute pathname
```

Comment:
Can you update the `args` description to include the arguments `recursive`, `abspath`, `non_deterministic`? Maybe length too?

Reply:
`non_deterministic: if False, the results yielded from each root directory will be sorted`

Reply:
Update `mask` to `masks`